### PR TITLE
Update free-courses-id.md

### PR DESCRIPTION
**Pull request by candraw**
Added Linux Fundamentals Course in Indonesian

## What does this PR do?
Add resource(s)

## For resources
Linux

### Description
Linux Fundamentals Course in Indonesian

### Why is this valuable (or not)?
Because it contains about linux fundamentals for beginners

### How do we know it's really free?
Because you can access the course on youtube, so its free

### For book lists, is it a book? For course lists, is it a course? etc.
Is it a Course

## Checklist:
- [*] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [*] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [*] Include author(s) and platform where appropriate.
- [*] Put lists in alphabetical order, correct spacing.
- [*] Add needed indications (PDF, access notes, under construction).
- [*] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-id.md [hunk 1]
```diff
@@ -272,11 +272,11 @@
 * [Belajar Kubernetes untuk Pemula](https://www.youtube.com/playlist?list=PL-CtdCApEFH8XrWyQAyRd6d_CKwxD8Ime) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
 
 
-### Linux
+###
```
Comment (metabiswadeep) [inline courses/free-courses-id.md:275]:
```suggestion
### Linux
```

File: courses/free-courses-id.md [hunk 2]
```diff
@@ -272,11 +272,11 @@
 * [Belajar Kubernetes untuk Pemula](https://www.youtube.com/playlist?list=PL-CtdCApEFH8XrWyQAyRd6d_CKwxD8Ime) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
 
 
-### Linux
+### 
 
 * [Linux Pemula](https://www.youtube.com/playlist?list=PLACzo3mGgHK_D2wkGu4AvUAQiqLC30PTz) - LINUXcare
 * [Tutorial Terminal Linux Bahasa Indonesia](https://www.youtube.com/playlist?list=PLy1BHEa_Wr-cpjWY5uOqsMxQwiQrRnC8B) - ArtAway
-
+* [Belajar linux fundamentals](https://youtube.com/playlist?list=PLbLqbqNn7VYpnd7FggSeq18AgE4gdsy2F) - ID-Networkers
```
Comment (metabiswadeep) [inline courses/free-courses-id.md:280]:
```suggestion


```

File: courses/free-courses-id.md [hunk 3]
```diff
@@ -272,11 +272,11 @@
...
```
Comment (metabiswadeep) [inline courses/free-courses-id.md:279]: 279 -> 277

**Review by metabiswadeep — CHANGES_REQUESTED** [review]

File: courses/free-courses-id.md (hunk 1 above)
Comment (metabiswadeep) [reply courses/free-courses-id.md:275]: Do not change things unnecessarily.

File: courses/free-courses-id.md [hunk 4]
```diff
@@ -276,7 +276,7 @@
...
```
Comment (davorpa) [inline courses/free-courses-id.md:279]:
```suggestion
* [Belajar linux fundamentals](https://youtube.com/playlist?list=PLbLqbqNn7VYpnd7FggSeq18AgE4gdsy2F) - ID-Networkers
* [Linux Pemula](https://www.youtube.com/playlist?list=PLACzo3mGgHK_D2wkGu4AvUAQiqLC30PTz) - LINUXcare
* [Tutorial Terminal Linux Bahasa Indonesia](https://www.youtube.com/playlist?list=PLy1BHEa_Wr-cpjWY5uOqsMxQwiQrRnC8B) - ArtAway
```

File: courses/free-courses-id.md (hunk 3 above)
Comment (davorpa) [reply courses/free-courses-id.md:279]: As suggested by @metabiswadeep 

```suggestion
* [Belajar linux fundamentals](https://youtube.com/playlist?list=PLbLqbqNn7VYpnd7FggSeq18AgE4gdsy2F) - ID-Networkers
* [Linux Pemula](https://www.youtube.com/playlist?list=PLACzo3mGgHK_D2wkGu4AvUAQiqLC30PTz) - LINUXcare
* [Tutorial Terminal Linux Bahasa Indonesia](https://www.youtube.com/playlist?list=PLy1BHEa_Wr-cpjWY5uOqsMxQwiQrRnC8B) - ArtAway
```

**Review by davorpa — APPROVED** [review]

**Pull request merged**